v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/model/helpers/package-info.java

```diff
@@ -0,0 +1,9 @@
+/**
+ * This module includes common helper objects shared between providers.
```

Comment:
package-info in all packages should include Apache 2.0 License. I'm surprised spotless didn't add this automatically. Did you run spotless:apply  on these files?